src/extensions/uv-seadragon-extension/SettingsDialogue.ts

```diff
@@ -61,6 +77,20 @@ class SettingsDialogue extends BaseSettingsDialogue {
 
             this.updateSettings(settings);
         });
+        
+        this.$clickToZoomEnabledCheckbox.change(() => {
+            var settings: ISettings = {};
+
+            if(this.$clickToZoomEnabledCheckbox.is(":checked")) {
+                settings.clickToZoomEnabled = true;
+            } else {
+                settings.clickToZoomEnabled = false;
+            }
+
+            this.updateSettings(settings);
+            var viewer = (<ISeadragonExtension>this.extension).getViewer();
```

Comment:
SeadragonCenterPanel should listen for the SETTINGS_CHANGED event and update the gestureSettings then.

https://github.com/UniversalViewer/universalviewer/blob/master/src/modules/uv-shared-module/BaseCommands.ts#L68

e.g. https://github.com/UniversalViewer/universalviewer/blob/cf99a78f10509ed94ba56746866b0a0d53ecd2ce/src/modules/uv-shared-module/FooterPanel.ts#L36